templates/compose/twenty.yaml

```diff
@@ -0,0 +1,46 @@
+# documentation: https://docs.twenty.com
+# slogan: Twenty is a CRM designed to fit your unique business needs.
+# tags: crm, self-hosted, dashboard
+# logo: svgs/twenty.svg
+
+services:
+  twenty:
+    image: 'twentycrm/twenty:latest'
+    environment:
+      SERVER_URL: $SERVICE_FQDN_TWENTY_3000
+      FRONT_BASE_URL: $SERVICE_FQDN_TWENTY_3000
+      ENABLE_DB_MIGRATIONS: true
+      SIGN_IN_PREFILLED: true
+      STORAGE_TYPE: local
```

Comment:
This could be problematic. https://docs.twenty.com/start/self-hosting/#storage, you would want this to be `local` or `s3` depending on the end users config.